src/logger/logger.ts

```diff
@@ -6,6 +6,12 @@ import * as winston from "winston";
 import TelemetryTransport from "./telemetry-transport";
 import UserNotificationTransport from "./user-notification-transport";
 
+export enum LogLevel {
+    info = "info",
```

Comment:
Please follow the typescript [coding guidelines](https://github.com/microsoft/TypeScript/wiki/Coding-guidelines#names) and use PascalCase for enum values.

Reply:
Updated enum values to PascalCase